### ISSUE TITLE
old nodes import the parser module (deprecated)

### ISSUE THREAD

**Issue by zeffii** [nortikin/sverchok#4505]
what to do.. replace with `ast.parse()`  ?

**Comment by Durman** [comment]
If we'd know for sure that it does not change its behavior it's worth to replace.
If it does change, it can break layouts and users wont be able to understand  the cause of it.

In the last case probably it would be better to remove the node implementation and show the message that, for example, Sverchok 1.0 was the last version where the node was supported. And we can wait till the deprecated module will be removed before doing this.

**Comment by zeffii** [comment]
in py3.10 `parser` is gone from standard libs. It is not a big deal, the nodes don't use the module extensively..

in essence they use these features:
```python
string_formula = 'x*n[0]'
x = 10
n = [20, 30]
```
we use
```python
import parser
code_formula = parser.expr(string_formula).compile()
k = eval(code_formula)

print(k)
# >>> 200
```
equivalent is 
```python
# without parser module
result = compile(string_formula, '<string>', mode='eval')
k = eval(result)

print(k)
# >>> 200
```

**Comment by zeffii** [comment]
a replacement could be
```python
from dataclasses import dataclass
try:
    import parser
except:
    
    @dataclass
    class Expr(str):
        str_formula: str
        def compile(self):
            print('use mock')
            return compile(self.str_formula, '<string>', mode='eval')

    parser = lambda: None
    parser.expr = lambda str_formula: Expr(str_formula)
```

**Comment by zeffii** [comment]
resolved: #4509
